### ISSUE TITLE
create_diff is unreasonably slow

### ISSUE THREAD

**Issue by hchataing** [google/googletest-rust#250]
Diffing files of ~500 lines with minor differences takes more than a couple minutes, which makes it practically unusable for interactive use.

**Comment by bjacotg** [comment]
Which version are you using? Normally, with the latest (0.8.x), #224 should have improved the performance of create_diff.

**Comment by hchataing** [comment]
The version was locked at 0.7, let me try again with 0.8.

**Comment by hchataing** [comment]
Much better ! sorry for not noticing the upgrade :)
